src/Payments.Jobs.MoneyMovement/Payments.Jobs.MoneyMovement.csproj

```diff
@@ -8,18 +8,16 @@
     <FileVersion>1.0.0.1</FileVersion>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Content Include="appsettings.json">
-      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
-    </Content>
-  </ItemGroup>
-
   <ItemGroup>
     <ProjectReference Include="..\Payments.Core\Payments.Core.csproj" />
     <ProjectReference Include="..\Payments.Jobs.Core\Payments.Jobs.Core.csproj" />
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="appsettings.json">
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
I don't recognize this change - is this the standard way to copy json files now?

Reply:
That's how Visual Studio sets it in other projects. For some reason this one was not set, and so local debugging of job wasn't picking up settings.